protocol/app/upgrades/v5.0.0/upgrade.go

```diff
@@ -204,6 +206,39 @@ func initializeOIMFCaps(
 	}
 }
 
+func voteExtensionsUpgrade(
+	ctx sdk.Context,
+	keeper consensusparamkeeper.Keeper,
+) {
+	currentParams, err := keeper.Params(ctx, &consensustypes.QueryParamsRequest{})
+	if err != nil {
+		panic(fmt.Sprintf("failed to retrieve existing consensus params in VE upgrade handler: %s", err))
+	}
+	if currentParams.Params.Abci.VoteExtensionsEnableHeight != 0 {
+		panic(fmt.Sprintf(
+			"unable to update VE Enable Height since its current value of %d is already non-zero",
+			currentParams.Params.Abci.VoteExtensionsEnableHeight))
+	}
+	veEnableHeight := ctx.BlockHeight() + 1
```

Comment:
I would prefer to have the enabled height be buffered with a few heights in between. Given that comet and cosmos have delayed updates for things like val-sets, it may also be the the case the consensus params are delayed in how quickly they are propagated.

Something like `ctx.BlockHeight() + 4/5`

Reply:
Could you elaborate on this, what is the difference between using currentHeight + 1 versus currentHeight + 5